### DISCUSSION TITLE
Where is the sites list stored?

### DISCUSSION THREAD

**Discussion by warpdesign** [GetPublii/Publii#1371] [Help] [answered]
I am trying to find a way to edit a website from several computers/locations. For that I first created a site on one computer then put the `sites`, `plugins`, `themes` into a git repository (I could not copy the config files because it seems to have settings specific to the local computer, like some paths).

I then installed Publii on another computer and then fetched these directories and put them inside my Publii user directory.

When running Pubii, it doesn't detect any existing site and allows me to create a new one. So I'm guessing there is a list of known sites that is stored somewhere but could not find where.

So I have some questions:
- where is stored the list of current sites?
- are files from an OS compatible with other OS (I created the site on Windows and am attempting to open it on a mac)?

**Comment by bobmitro** [comment]
Refer to this doc: https://getpublii.com/docs/install-publii.html

**Comment by dziudek** [comment]
Sites list is built from the sites location directory - so on different computers it will be different - most probably you won't have the same sites directory path on different computers.

Site files are compatible between OSes.

**Comment by warpdesign** [comment]
I know the sites location will be different in computers. My question is: once I found the sites location on a computer, if I add manually a site I copied from another computer, is it supposed to be recognized once I launch Publii? Because that's what I did, and it was not recognized :/

**Comment by dziudek** [comment, marked as answer]
@warpdesign - if you put the site in the folder set in your Publii instance as sites directory, then it should be automatically recognized. Alternatively you can create a backup of the website and add it on another instance using install from backup option on the create website view

**Reply by warpdesign** [reply]
I wasn't comiting the sites' config directory: I wrongly added config/ into .gitignore which acted as a glob pattern so not only ignored the Publii's config directory, but also each site's config directory.

Now it's working. Only problem is that every site's config seems to contain some local path as well, like the outputDirectory in site.config.json.

Still, that's a good start!

**Reply by dziudek** [reply]
@warpdesign - I have some idea how to make it even better in v.0.45. I will come back when I will prepare some PoC :)
